### ISSUE TITLE
Remove @ExtendWith(SpringExtension.class) when @SpringBootTest is present

### ISSUE THREAD

**Issue by pway99** [openrewrite/rewrite-spring#43]
As of SpringBoot 2.1.0.M1 the `@SpringBootTest` is annotated with `@ExtendWith(SpringExtension.class)`.

If the `@SpringBootTest` annotation is present then the `SpringRunnerToSpringExtension`recipe should just remove the `@RunsWith(SpringRunner)` annotation.

**Comment by tedyoung** [comment]
I just ran into this issue in 4.5.0, it replaced `@RunsWith` with `@ExtendWith` on a test annotated with `@SpringBootTest` after the `org.openrewrite.java.spring.boot2.SpringBoot1To2Migration` recipe ran.

Before:

```
@RunWith(SpringRunner.class)
@SpringBootTest
@AutoConfigureMockMvc
@ActiveProfiles("test")
public class MealOrderApiIntegrationTest {
```

after

```
@ExtendWith(SpringExtension.class)
@SpringBootTest
@AutoConfigureMockMvc
@ActiveProfiles("test")
public class MealOrderApiIntegrationTest {
```

As stated above, the `@ExtendWith` is no longer needed.

**Comment by jkschneider** [comment]
@tkvangorder is this handled by https://github.com/openrewrite/rewrite-spring/issues/72?

**Comment by tkvangorder** [comment]
This was an oversight, we have the recipe (`org.openrewrite.java.spring.boot2.UnnecessarySpringExtension`) to remove the redundant `@ExtendWith(SpringExtension.class)`, it is just not included in the composite `SpringBoot1To2Migration`

**Comment by tkvangorder** [comment]
This has been fixed with 2b93850e367e8f74cd14f8a16903118cf6da1927 and is part of the v4.6.1 release.
